examples/kalthoff_winkler.cpp

```diff
@@ -18,126 +18,129 @@
 
 #include <CabanaPD.hpp>
 
+// Simulate the Kalthoff-Winkler experiment of crack propagation in
+// a pre-notched steel plate due to impact.
+void kalthoffWinklerExample( const std::string filename )
+{
+    // ====================================================
+    //             Use default Kokkos spaces
+    // ====================================================
+    using exec_space = Kokkos::DefaultExecutionSpace;
+    using memory_space = typename exec_space::memory_space;
+
+    // ====================================================
+    //                   Read inputs
+    // ====================================================
+    CabanaPD::Inputs inputs( filename );
+
+    // ====================================================
+    //                Material parameters
+    // ====================================================
+    double rho0 = inputs["density"];
+    double E = inputs["elastic_modulus"];
+    double nu = 1.0 / 3.0;
```

Comment:
@streeve We may include the value of the Poisson's ratio in the inputs, for consistency.

Reply:
Can you include in a separate PR (either with the unused variables or separate)?